Project/RAFinder/app/src/main/java/edu/rosehulman/rafinder/controller/student/StudentProfileFragment.java

```diff
@@ -21,48 +18,41 @@
  */
 public class StudentProfileFragment extends Fragment {
     private StudentProfileListener mListener;
-    private Resident resident;
+    private Employee resident;
+
     public StudentProfileFragment() {
     }
 
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        if (mListener!=null){
-            resident=mListener.getSelectedResident();
+        if (mListener != null) {
+            resident = mListener.getSelectedResident();
         }
     }
 
     public static StudentProfileFragment newInstance() {
         return new StudentProfileFragment();
     }
+
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
-        View view= inflater.inflate(R.layout.fragment_profile_student, container, false);
-        TextView nameTextView= (TextView) view.findViewById(R.id.nameTextView);
-        TextView roomTextView= (TextView) view.findViewById(R.id.roomTextView);
-        TextView locationTextView= (TextView) view.findViewById(R.id.locationTextView);
-        TextView statusTextView= (TextView) view.findViewById(R.id.statusTextView);
-        TextView phoneTextView= (TextView) view.findViewById(R.id.phoneTextView);
-        TextView emailTextView= (TextView) view.findViewById(R.id.emailTextView);
-        nameTextView.setText(resident.getName());
-        roomTextView.setText("Room: "+resident.getHall()+" "+resident.getRoom());
+        View view = inflater.inflate(R.layout.fragment_profile_student, container, false);
```

Comment:
Most of this is rearranging. statusDetailTextView is new.
Also, as I noted, duplicated code. Not sure if it's fixable, given that R is context-dependent.